guides/common/modules/con_configuring-project-to-manage-the-lifecycle-of-a-host-registered-to-a-freeipa-realm.adoc

```diff
@@ -45,11 +45,16 @@ Complete the following procedure on {Project} and every {SmartProxy} that you wa
 ----
 # scp /root/freeipa.keytab root@_{smartproxy-example-com}_:/etc/foreman-proxy/freeipa.keytab
 ----
-. Move the `/root/freeipa.keytab` file to the `/etc/foreman-proxy` directory and set the ownership settings to the `foreman-proxy` user:
+. On the {ProjectServer}, if you are configuring it for realm support, move the `/root/freeipa.keytab` file to the `/etc/foreman-proxy` directory:
```

Comment:
```suggestion
. On your {ProjectServer}, move the `/root/freeipa.keytab` file to the `/etc/foreman-proxy` directory:
```

suggestion (non-blocking): Can we drop this because it's a step under the "for {FreeIPA} realm support" heading from line 38?

Another suggestion: Use "your {ProjectServer}" in favor of "the {ProjectServer}".